packages/apps/src/microsoft/teams/apps/routing/activity_context.py

```diff
@@ -34,14 +34,10 @@
 from microsoft.teams.api.models.oauth import OAuthCard
 from microsoft.teams.cards import AdaptiveCard
 from microsoft.teams.common import Storage
+from msgraph.graph_service_client import GraphServiceClient
 
 from ..plugins import Sender
 
-if TYPE_CHECKING:
```

Comment:
We still need this...
I was referring to `GraphServiceClient = object` not being necessary since it's not used at runtime